plugins/Jobs.cpp

```diff
@@ -239,7 +239,7 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
                 }
 
                 // If there's no job or the existing job doesn't match the data we've been passed, escalate to master.
-                if (!result.empty() && result[0][1] == job["data"]) {
+                if (!result.empty() && result[0][1] == (job["data"].empty() ? SToStr("{}") : job["data"])) {
```

Comment:
`"{}"` was a different type than `SString`, it was like `char` or whatever and therefore comparison didn't work
No CPP pro here, you tell me if there's a better way to do this